File: problemas/1000/1015.dart
import 'dart:io';
import 'dart:math';

void main() {
  List entrada = stdin.readLineSync().split(' ');
  final x1 = double.parse(entrada[0]);
  final y1 = double.parse(entrada[1]);
  entrada = stdin.readLineSync().split(' ');
  final x2 = double.parse(entrada[0]);
  final y2 = double.parse(entrada[1]);

  final distancia = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));
  stdout.writeln(distancia.toStringAsFixed(4));
}

File: problemas/1000/1015.py
import math

entrada = input().split(' ')
x1 = float(entrada[0])
y1 = float(entrada[1])
entrada = input().split(' ')
x2 = float(entrada[0])
y2 = float(entrada[1])

distancia = math.sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2))
print('{:.4f}'.format(distancia))
